package/yast2-fonts.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Mar  2 12:40:29 UTC 2018 - antoine.belvire@opensuse.org
+
+- Fix paragraph end tag.
```

Comment:
please add bug number of FATE entry for this change. otherwise it looks good.

Reply:
There is no bug number nor fate entry.

Reply:
So there is no way to push something without bug number? Will a GitHub issue will do or do I have to use openSUSE's Bugzilla?

Reply:
well, in current phase of development we need opensuse bugzilla number in format `bsc#xxxxxxxx`. It is used to verify that change is synchronized in all products.

Reply:
I may be wrong. But I don't think yast2-fonts is included in all products (it's not in SLE, AFAIK). So the `bsc#` reference may not be mandatory in this case.

Reply:
[Weblate says it's only in Tumbleweed](https://l10n.opensuse.org/projects/yast-fonts/master/fr/). I only do this because I believe this `<\p>` triggers unwanted .po(t) update – sometimes it's `<\p>`, sometimes it's `<p>` in the string to translate. Anyway I will open a bug report if you prefer.